### ISSUE TITLE
Commit 549206 in PR #2929 broke KokkosKernels build

### ISSUE THREAD

**Issue by brian-kelley** [kokkos/kokkos#2983]
@DavidPoliakoff I think there's an issue with include paths:

```
[ 19%] Building CXX object src/CMakeFiles/kokkoskernels.dir/impl/generated_specializations_cpp/nrm2/Blas1_nrm2_mv_eti_DOUBLE_LAYOUTLEFT_EXECSPACE_OPENMP_MEMSPACE_HOSTSPACE.cpp.o
In file included from /home/bmkelle/Fix703-testing-1588019954/Testing/TestAll_2020-04-27_13.39.20/gcc/7.3.0/OpenMP-release/kokkos-install/include/impl/Kokkos_Profiling_Interface.hpp:89:0,
                 from /home/bmkelle/Fix703-testing-1588019954/Testing/TestAll_2020-04-27_13.39.20/gcc/7.3.0/OpenMP-release/kokkos-install/include/impl/Kokkos_Profiling.hpp:48,
                 from /home/bmkelle/Fix703-testing-1588019954/Testing/TestAll_2020-04-27_13.39.20/gcc/7.3.0/OpenMP-release/kokkos-install/include/impl/Kokkos_Tools.hpp:52,
                 from /home/bmkelle/Fix703-testing-1588019954/Testing/TestAll_2020-04-27_13.39.20/gcc/7.3.0/OpenMP-release/kokkos-install/include/Kokkos_View.hpp:59,
                 from /home/bmkelle/Fix703-testing-1588019954/Testing/TestAll_2020-04-27_13.39.20/gcc/7.3.0/OpenMP-release/kokkos-install/include/Kokkos_Parallel.hpp:53,
                 from /home/bmkelle/Fix703-testing-1588019954/Testing/TestAll_2020-04-27_13.39.20/gcc/7.3.0/OpenMP-release/kokkos-install/include/Kokkos_OpenMP.hpp:62,
                 from /home/bmkelle/Fix703-testing-1588019954/Testing/TestAll_2020-04-27_13.39.20/gcc/7.3.0/OpenMP-release/kokkos-install/include/Kokkos_Core.hpp:58,
                 from /ascldap/users/bmkelle/Fix703-testing-1588019954/kokkos-kernels/src/blas/impl/KokkosBlas1_scal_spec.hpp:48,
                 from /ascldap/users/bmkelle/Fix703-testing-1588019954/Testing/TestAll_2020-04-27_13.39.20/gcc/7.3.0/OpenMP-release/src/impl/generated_specializations_cpp/scal/Blas1_scal_mv_eti_COMPLEX_DOUBLE_LAYOUTLEFT_EXECSPACE_OPENMP_MEMSPACE_HOSTSPACE.cpp:48:
/home/bmkelle/Fix703-testing-1588019954/Testing/TestAll_2020-04-27_13.39.20/gcc/7.3.0/OpenMP-release/kokkos-install/include/impl/Kokkos_Profiling_DeviceInfo.hpp:49:10: fatal error: impl/Kokkos_Profiling_C_Interface.h: No such file or directory
 #include <impl/Kokkos_Profiling_C_Interface.h>
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```
but I'm not sure why. ``core/src/impl/Kokkos_Profiling_C_Interface.h`` clearly exists so this error seems to imply that core/src is not in the include path. It's not a compiler bug though, it's failing on all the KokkosKernels spot-check builds (GCC, Intel, ...).

**Comment by DavidPoliakoff** [comment]
Brian, I'd bet I failed to install that file or something. Let me check, sorry about that

**Comment by DavidPoliakoff** [comment]
Yeah, I did. PR incoming soon

**Comment by stanmoore1** [comment]
ExaMiniMD is still failing with
```make: *** No rule to make target `Kokkos_Profiling.o', needed by `libkokkos.a'.  Stop.```

**Comment by DavidPoliakoff** [comment]
I must have forgotten the Makefile based build system we have. Sorry Stan, that's next on my list

**Comment by DavidPoliakoff** [comment]
@stanmoore1 : Thanks for letting me know, #2985 is up to fix it. Sorry about that

**Comment by stanmoore1** [comment]
No worries, thanks

**Comment by brian-kelley** [comment]
@DavidPoliakoff Thanks for fixing this quickly.

**Comment by DavidPoliakoff** [comment]
Closed via #2984 and #2985

**Comment by DavidPoliakoff** [comment]
Weird, the issue showed as open for me. Thanks for closing it Brian
